bindings/python/iota_sdk/types/event.py

```diff
@@ -8,16 +8,14 @@ class WalletEventType(IntEnum):
     """Types of wallet events.
 
     Attributes:
-        ConsolidationRequired (0): Consolidation is required.
-        LedgerAddressGeneration (1): Nano Ledger has generated an address.
-        NewOutput (2): A new output was created.
-        SpentOutput (3): An output was spent.
-        TransactionInclusion (4): A transaction was included into the ledger.
-        TransactionProgress (5): A progress update while submitting a transaction.
+        LedgerAddressGeneration (0): Nano Ledger has generated an address.
+        NewOutput (1): A new output was created.
+        SpentOutput (2): An output was spent.
+        TransactionInclusion (3): A transaction was included into the ledger.
```

Comment:
Should this be finalized now? Or do we maybe need confirmed and finalized?

Reply:
Let's keep it like this for now, added a comment to https://github.com/iotaledger/iota-sdk/issues/1814